editors/intellij/README.md

```diff
@@ -0,0 +1,44 @@
+# Biome JetBrains IDEs Plugin
+
+https://biomejs.dev plugin for JetBrains IDEs.
+
+## features
+
+- Linting
+- Quickfix
+- Formatting
+
+---
+
+## Installation
+
+### From JetBrains IDEs
+
+- Press `⌘Сmd,` to open the IDE settings and then select Plugins.
+![Plugins](https://resources.jetbrains.com/help/img/idea/2023.2/ws_plugins_settings.png)
+- Search for Biome and click `install`
+
+### From disk
+
+- Download the plugin .zip from releases tab.
+- Press `⌘Сmd,` to open the IDE settings and then select Plugins.
+- On the Plugins page, click The Settings button and then click Install Plugin from Disk….
+
+## Building and running the plugin
+
+Build and run the plugin requires:
+
+- Java development kit 17+
+- IntelliJ IDEA (Ultimate edition or community edition)
```

Comment:
Is is this required for **installing** the plugin? I think it's only required for local development. Is there a minimum version for the IntelliJ IDEs?

Reply:
You are right, this requirements are only needed for development, i think we can safely remove this, because it already exists on CONTRIBUTING.md.

About minimum version, the plugin template configure the minimum version to 2023.2, but we can easily change that on gradle.properties file
(Maybe we need to add the [plugin version verifier](https://plugins.jetbrains.com/docs/intellij/verifying-plugin-compatibility.html#plugin-verifier) to the github workflow, to ensure the biome plugin binary compatibility)

[For more detail about version ranges](https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html)